docs/developers-clear-containers-install.md

```diff
@@ -85,9 +86,34 @@ to allow their creation.
 
 ```bash
 $ sudo yum install -y glibc-static
-$ cd $GOPATH/src/github.com/clearcontainers/runtime/.ci/
+$ cd $GOPATH/src/github.com/clearcontainers/tests/.ci/
 $ ./install_virtcontainers.sh
 ```
+Don't forget to adjust pause_root_path in /usr/local/etc/clear-containers/configuration.toml
```

Comment:
This reminder (and the one below) can be removed if the runtime is built like this:

- https://github.com/clearcontainers/tests/blob/master/.ci/install_runtime.sh#L29..L42